templates/jobs/base.html

```diff
@@ -0,0 +1,40 @@
+{% load staticfiles %}
+<!DOCTYPE HTML>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" >
+  <title>{% block title %}Django Girls - start your journey with programming{% endblock %}</title>
+  <meta name="description" content="Django Girls is a one-day workshop about programming in Python and Django for women.">
+  <meta name="keywords" content="programming, python, django, workshop, women, females">
+  <link href="{% static 'img/favicon.ico' %}" rel="shortcut icon">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta property="og:title" content="Django Girls is a one-day workshop about programming in Python and Django for women" />
+  <meta property="og:type" content="website" />
+  <meta property="og:description" content="Join us and become female developer! Build your first website in Python and Django in a friendly, safe enviroment." />
+  <meta property="og:url" content="http://djangogirls.org/" />
+  <meta property="og:image" content="https://raw.githubusercontent.com/olasitarska/djangogirls/gh-pages/resources/graphics/logo_square.png" />
+
+  <link rel="stylesheet" href="//brick.a.ssl.fastly.net/Linux+Libertine:400,600,700,400i,600i,700i/Roboto:300,400,300i,400i">
+  <link rel="stylesheet" type="text/css" href="{% static 'vendor/css/bootstrap.min.css' %}" media="screen">
+  <link rel="stylesheet" type="text/css" href="{% static 'vendor/css/sss.css' %}" media="screen">
+  <link rel="stylesheet" type="text/css" href="{% static 'css/global.css' %}" media="screen">
+  <script type="text/javascript" src="{% static 'vendor/js/jquery.min.js' %}"></script>
+  <script type="text/javascript" src="{% static 'vendor/js/bootstrap.min.js' %}"></script>
+  <script type="text/javascript" src="{% static 'vendor/js/sss.min.js' %}"></script>
+  {% block extra_js %}{% endblock %}
+</head>
+<body>
+    <div class="page-wrap">
+        {% block menu %}
+            {% include 'jobs/menu.html' %}
+        {% endblock %}
+
+        {% block content %}{% endblock %}
+    </div>
+
+    {% block footer %}
+        {% include 'global/footer.html' %}
+    {% endblock %}
+
+    {% include 'includes/js.html' %}
+</body>
+</html>
```

Comment:
This could just extend global/base and replace the menu block.  Everything else looks identical.